cmake/FindDPCPP.cmake

```diff
@@ -38,7 +38,7 @@ find_library(DPCPP_LIB_DIR NAMES sycl sycl6 PATHS "${DPCPP_BIN_DIR}/../lib")
 
 add_library(DPCPP::DPCPP INTERFACE IMPORTED)
 
-set(DPCPP_FLAGS "-fsycl;")
+set(DPCPP_FLAGS "-fsycl;-mllvm;-enable-global-offset=false;")
```

Comment:
This flag was moved to line 58

```cmake
 list(APPEND DPCPP_COMPILE_ONLY_FLAGS; "-mllvm;-enable-global-offset=false;")
 ```

Reply:
There was a TODO comment which I thought I had added as a part of [0c9d5e1](https://github.com/codeplaysoftware/cutlass-fork/pull/126/commits/0c9d5e1844bc4669840a8fc2c1259b6656077d5e) over here,
which basically was basically about investigating why this line is needed, 

I was aware of this change, but for some reason I was still seeing a kernel `*_with_offset`, hence I added that as a temporary fix, 
This is also party the reason why this PR is draft